app/keepers/keepers.go

```diff
@@ -349,6 +349,7 @@ func NewAppKeeper(
 	// Set legacy router for backwards compatibility with gov v1beta1
 	govKeeper.SetLegacyRouter(govRouter)
 
+	// appKeepers.GovKeeper = *govKeeper
```

Comment:
Can we remove this ?
```suggestion
```

Reply:
Sry, missed the comment and merged. Will remove at a later time